frontend-collective/frontend-collective-react-dnd-scrollzone
# update-to-react-17

https://github.com/frontend-collective/frontend-collective-react-dnd-scrollzone/pull/80
Issues
Many people are having issues with react-sortable-tree after upgrading to React 17. It looks like the cause is using findDOMNode and mismatched React versions due to a peer dependency issue. This PR should resolve the issues.

https://github.com/facebook/react/issues/20131#issuecomment-762783107
https://github.com/frontend-collective/react-sortable-tree/issues/821
https://github.com/frontend-collective/react-sortable-tree/issues/450
https://github.com/frontend-collective/react-sortable-tree/issues/281#issuecomment-719389673
https://github.com/frontend-collective/frontend-collective-react-dnd-scrollzone/issues/70

Test steps
yarn
yarn build
in the react-sortable-tree repo
yarn add file:$PATH_TO_THIS_REPO/lib
add in some yarn resolutions in package.json so storybook will use React 17
"resolutions": {
    "react": "^17.0.1",
    "react-dom": "^17.0.1"
}
yarn storybook
test around make sure nothing is broken
